### PR TITLE
Incident displays: wallening edition

### PR DESCRIPTION
**Pull request by lessthnthree**
## About The Pull Request

Redoes the delam/tram incident displays to be wallening compatible.

image

## Why It's Good For The Game

Looks better, and won't be jank when wallening comes.

## Changelog

:cl: LT3
image: Incident displays got a makeover
/:cl:

### CODE REVIEW

File: code/game/machinery/incident_display.dm [hunk 1]
```diff
@@ -117,35 +146,22 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/machinery/incident_display/tram, 32)
 
 	if(sign_features == DISPLAY_TRAM)
 		tool.play_tool_sound(src)
-		balloon_alert(user, "set to dual")
-		name = NAME_DUAL
-		desc = DESC_DUAL
-		icon_state = "stat_display_dual"
-		sign_features = DISPLAY_DELAM | DISPLAY_TRAM
+		balloon_alert(user, "set to delam")
```
Comment (OrionTheFox) [inline code/game/machinery/incident_display.dm:149]: Might be a bit out of scope, but does this interaction have an examine/tooltip yet so players know multitools can do this? Same question goes for the welder and repairs

Comment (lessthnthree) [reply code/game/machinery/incident_display.dm:149]: I'll double check. I think maybe? If not I'm fine with adding one.

Comment (lessthnthree) [reply code/game/machinery/incident_display.dm:167]: Done.

**Comment by Jacquerel** [comment]
I like the sprite but I don't know if it looks great on our pre-wallening walls, are we close enough that we want to merge it anyway?

**Review by Wallemations — APPROVED** [review]

**Comment by norsvenska** [comment]
i love the adverts
how simple is it to add new ones? i've some ideas

**Comment by lessthnthree** [comment]
> i love the adverts how simple is it to add new ones? i've some ideas

It's quite easy, we were discussing it on the discord earlier about the possibilities. A later PR will make it even easier (I am going to move the adverts to their own .dmi and such, so people can just add new icon states.

**Comment by san7890** [comment]
> I like the sprite but I don't know if it looks great on our pre-wallening walls, are we close enough that we want to merge it anyway?

we should be fine for 3 weeks. better now than have it be in the fuckhuge pr

**Pull request merged**